test/test_jit.py

```diff
@@ -4629,16 +4631,21 @@ def __init__(self):
                 self.a = 1
                 self.b = 1.2
                 self.c = False
-                self.d = [nn.Linear(3, 4)]
+                with tester.assertRaisesRegex(
+                        TypeError,
+                        "'Linear' object for attribute 'd' is not a valid constant"):
+                    self.d = [nn.Linear(3, 4)]
                 self.e = lambda x: x
                 self.f = [3, 4, 5]
-                self.assertTrue(type(self.f) is tuple)
+                tester.assertTrue(type(self.f) is tuple)
                 self.g = [3, (3, 4), 5]
-                with self.assertRaisesRegex(TypeError, "is not a valid constant"):
+                with tester.assertRaisesRegex(TypeError, "not a valid constant"):
                     self.h = type(1)
-                with self.assertRaisesRegex(TypeError, "is not a valid constant"):
+                with tester.assertRaisesRegex(TypeError, "not a valid constant"):
                     self.i = (3, 4, {})
 
+        f = Foo()
```

Comment:
Why does `Foo()` need to be constructed?

Reply:
The `ScriptModule.__init__()` needs to run to actually compile the methods, before this test was a no-op